2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -214,13 +215,149 @@ Viene inserito ";" alla fine delle voci dell'elenco tranne l'ultima che termina
++ esecuzione (Do): attuazione del piano;
++ verifica (Check): monitoraggio e misurazione dei processi e dei prodotti rispetto agli obiettivi e ai requisiti, e reportistica dei risultati;
++ attuazione (Act): azioni per migliorare le prestazioni, se necessario, in base ai risultati della verifica.
+
+==== Organizzazione del processso 
+===== Pianificazione
+La pianificazione del processo è compito del responsabile, il quale al termine dello sprint precedente, in base alle attività svolte e ai risultati ottenuti, pianifica le attività da svolgere nello sprint successivo. \
+La pianificazione è un'attività iterativa, che viene svolta ad ogni termine di sprint. Il responsabile, in relazione al progresso svolto, può decidere di modificare la pianificazione iniziale, aggiungendo o rimuovendo attività. \
+Questo processo permette di individuare le attività da svolgere, le risorse necessarie e le tempistiche di svolgimento, mediante il sistema di ticketing offerto da Jira.\
+Il risutato di questo processo è visibile all'interno del Piano di Progetto.\
+\ 
+===== Esecuzione
+Identifica il processo di sviluppo del prodotto, dove quanto pianificato viene concretamente svolto. \
+Il processo di esecuzione è composto da due attività principali:
++ sviluppo del prodotto;
++ sviluppo della documentazione.
+
+Durante questo processo, ogni ruolo svolge le attività assegnate seguendo quanto stabilito nella pianificazione, e sarà compito del responsabile verificare che le attività siano svolte correttamente e nei tempi previsti. Ogni membro avrà la possibilità di segnalare eventuali criticità, avendo a disposizione una board apposita sulla piattaforma Miro.\ Si tratta di un passaggio fondamentale in quanto permette di avere un resoconto pronto a fine sprint in merito al processo di avanzamento, individuando: \
+- "Keep doing": attività che hanno dato buoni risultati e che quindi vanno mantenute;
+- "Things to change": attività che hanno dato risultati non soddisfacenti e che quindi vanno modificate.
+\
+La progressione del lavoro è visibile mediante:
+- grafici di burndown (Jira);
+- board di avanzamento (Miro).
+
+I prodotti di questo processo, permettono dunque di procedere con la verifica e lo stabilimento delle contromisure e dei miglioramenti necessari.
+\ \
+===== Verifica
+Al termine di ogni sprint, il gruppo procederà con il meeting di retrospettiva, durante il quale verranno analizzati i risultati ottenuti e le attività svolte, basandosi sui prodotti della fase di esecuzione: \
++ grafici di burndown:
+  - Permetto di avere una visione rapida di quanto del lavoro pianificato è stato portato a termine: si tratta di una metrica puramente quantitativa, che tiene conto di del numero di ticket chiusi e del numero di ticket aperti;
++ board di avanzamento:
+  - "Keep doing": attività che hanno dato buoni risultati e che quindi vanno mantenute: questo permette al gruppo di individuare il modus operandi più efficace ed efficiente per svolgere le attività;
```

Comment:
modus operandi mi piace ma lo metterei in corsivo forse(?) lascio a te la decisione